docs/ecosystem.md

```diff
@@ -5,22 +5,30 @@ This document tracks the crates that already exist in the ecosystem that cover f
+|                         | [rust_icu_uloc](https://crates.io/crates/rust_icu_uloc)        | filmil, kpozin      | Yes              | Uncertain  | Rust wrapper around ICU4C                                          |
+| `icu::Bidi`             | [unicode-bidi](http://docs.rs/unicode-bidi)                    | Servo / Manish      | N/A              | No Action  | Bidi and text layout are unlikely to be coming to ECMA-402         |
+| `icu::Normalization`    | [unicode-normalization](http://docs.rs/unicode-normalization/) | Manish / unicode-rs | No               | Uncertain  | Main issue: [#40](https://github.com/unicode-org/omnicu/issues/40) |
+| `icu::Script`           | [unicode-script](http://docs.rs/unicode-script/)               | Manish              | N/A              | No Action  | ICU4X should expose UCD data through its own custom pipeline       |
+| `icu::IDNa`             | [idna](http://docs.rs/idna/)                                   | Servo / Manish      | CLDR confusables | Uncertain  | Main issue: [#42](https://github.com/unicode-org/omnicu/issues/42) |
+| `icu::PluralRules`      | [intl-pluralrules](https://github.com/zbraniecki/pluralrules)  | Zibi                | Yes              | Import     |                                                                    |
+|                         | [rust_icu_intl][1]                                             | filmil, kpozin      | Yes              | Uncertain  |                                                                    |
+| `icu::BreakIterator`    | [unicode-segmentation](https://docs.rs/unicode-segmentation/)  | Manish / unicode-rs | No               | No Action  | No line segmentation. Wait for clear user demand outside Rust      |
+| `icu::Collator`         | -                                                              |                     |                  |            |                                                                    |
+| `icu::NumberFormatter`  | [Early Google POC](https://github.com/sffc/rust-wasm-i18n)     | Shane               | ?                | Import     |                                                                    |
+| `icu::CharConversion`   | -                                                              |                     |                  |            |                                                                    |
+| `icu::Char`             | -                                                              |                     |                  |            |                                                                    |
+|                         | [rust_icu_ustring](https://crates.io/crates/rust_icu_ustring)  | filmil, kpozin      | Yes              | Uncertain  |                                                                    |
```

Comment:
Why two lines for `icu::Char` when there is only one entry?

Reply:
Done.